packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -53,48 +67,64 @@ export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     }, [props.annotation, annotationValues, filtersForAnnotation]);
 
     const onDeselectAll = () => {
+        // remove all regular filters for this annotation
         dispatch(selection.actions.removeFileFilter(filtersForAnnotation));
     };
 
     const onDeselect = (item: ListItem) => {
-        const fileFilter = new FileFilter(
-            props.annotation.name,
-            isNil(props.annotation.valueOf(item.value))
-                ? item.value
-                : props.annotation.valueOf(item.value)
-        );
-        dispatch(selection.actions.removeFileFilter(fileFilter));
+        dispatch(selection.actions.removeFileFilter(createFileFilter(item)));
     };
 
     const onSelect = (item: ListItem) => {
-        const fileFilter = new FileFilter(
+        dispatch(selection.actions.changeFileFilterType(props.annotation.name, FilterType.DEFAULT));
+        dispatch(selection.actions.addFileFilter(createFileFilter(item)));
+    };
+
+    // TODO: Should this select ALL or just the visible items in list?
+    const onSelectAll = () => {
+        dispatch(selection.actions.changeFileFilterType(props.annotation.name, FilterType.DEFAULT));
+        dispatch(selection.actions.addFileFilter(items.map((item) => createFileFilter(item))));
+    };
+
+    const createFileFilter = (item: ListItem) => {
+        return new FileFilter(
             props.annotation.name,
             isNil(props.annotation.valueOf(item.value))
                 ? item.value
-                : props.annotation.valueOf(item.value)
+                : props.annotation.valueOf(item.value),
+            filterType
         );
-        dispatch(selection.actions.addFileFilter(fileFilter));
     };
 
-    const onSelectAll = () => {
-        const filters = items.map(
-            (item) =>
-                new FileFilter(
-                    props.annotation.name,
-                    isNil(props.annotation.valueOf(item.value))
-                        ? item.value
-                        : props.annotation.valueOf(item.value)
-                )
-        );
-        dispatch(selection.actions.addFileFilter(filters));
+    const _onFilterTypeOptionChange = (option: IChoiceGroupOption | undefined) => {
```

Comment:
truly don't remember, I think it might have been legacy. Removed in [d685394](https://github.com/AllenInstitute/biofile-finder/pull/238/commits/d685394f5e7800e206f6cada8585aee6a5d712d9)